README.md

```diff
@@ -8,7 +8,9 @@ This is a very rough prototype and is not ready for production use.
 ### Testnet
 The easiest way to run these scripts is with `dapp testnet`. Use this command to start a geth devserver.
 
+
 ### Pre-deploy:
+All deployed addresses are written to `addresses-$(seth chain).json`. To start with a fresh deploy, remove the address file.
```

Comment:
```
{
  "NFT_COLLATERAL": "0x...",
  "GOD_ADDR": "0x...",
  "CURRENCY": "0x...",
```
Is needed otherwise it would fail

Reply:
No, those can also  be exported as environment variables, they don't need to be in the file. Why do we need NFT_COLLATERAL?